server/meshmodel/meshery-core/0.7.2/v1.0.0/policies/alias_policy.rego

```diff
@@ -35,66 +49,29 @@ package relationship_evaluation_policy
 #        - Remove the alias component from the design file.
 #        - Remove the aliased configuration from the parent component.
 
-import rego.v1
-
 is_alias_relationship(relationship) if {
 	lower(relationship.kind) == "hierarchical"
 	lower(relationship.type) == "parent"
 	lower(relationship.subType) == "alias"
 }
```

Comment:
It is unlikely, that Meshery has a use case for supporting relationship.type == "child" aliases in the future.